at_client/lib/src/client/at_client_impl.dart

```diff
@@ -196,9 +195,9 @@ class AtClientImpl implements AtClient {
     var builder = DeleteVerbBuilder()
       ..isCached = isCached
       ..isPublic = isPublic
-      ..sharedWith = sharedWith
+      ..sharedWith = sharedWith!
```

Comment:
delete:phone@alice is a valid command. sharedWith will be null and will fail here